cypress/e2e/data-import/log-details/textlink-color-change.cy.js

```diff
@@ -0,0 +1,52 @@
+import { DevTeams, Permissions, TestTypes } from '../../../support/dictionary';
+import TopMenu from '../../../support/fragments/topMenu';
+import Users from '../../../support/fragments/users/users';
+import DataImport from '../../../support/fragments/data_import/dataImport';
+import Logs from '../../../support/fragments/data_import/logs/logs';
+import FileDetails from '../../../support/fragments/data_import/logs/fileDetails';
+
+describe('data-import', () => {
+  describe('Log details', () => {
+    let user;
+
+    before('create user and login', () => {
+      cy.createTempUser([Permissions.moduleDataImportEnabled.gui]).then((userProperties) => {
+        user = userProperties;
+      });
+    });
+
+    after('delete user', () => {
+      cy.getAdminToken();
+      Users.deleteViaApi(user.userId);
+    });
+
+    it(
+      'C353974 Check: Change Import log hotlinks to textLink: Landing page (folijet) (TaaS)',
+      { tags: [TestTypes.extendedPath, DevTeams.folijet] },
+      () => {
+        // #1 Go to the "Data Import" app
+        // User is on the Data Import landing page
+        cy.login(user.username, user.password, {
```

Comment:
I'd recommend to move login action to precondition

Reply:
fixed